docs/manuals/auth/index.md

```diff
@@ -0,0 +1,77 @@
+!!! warning "Reminder: Authorization and Authentication is only available in the Enterprise version"
+
+# Pre-requisites
+
+An LDAP server must be set up by the user. Taipy provides no support for setting up the server.
+
+# Configuration
+
+You must configure how Taipy connects to the LDAP server using the Taipy `Config^` features. Here are the configurable properties you can use:
+
+- **Config.global_config.ldap_server**: The url of the LDAP server.
+- **Config.global_config.ldap_base_dn**: The base DN that the LDAP server uses to search for users authentication.
+- **Config.global_config.secret_key**: The secret key that will be used to generate the access token
+- **Config.global_config.auth_session_duration**: The period after which the token provided by Taipy will be expired.
+- **Config.global_config.auth_enabled**: The field to enable or disable Taipy authentication and authorization. Its default value is True, meaning that the enterprise version, by default, will have authentication and authorization feature enabled.
+
+Here is an example of configuring LDAP server connection in Taipy:
+
+
+=== "Python configuration"
+
+    ```
+    Config.configure_global_app(ldap_server="ldap://0.0.0.0",
+                                ldap_base_dn="dc=example,dc=org",
+                                secret_key = "my-ultra-secure-and-ultra-long-secret",
+                                auth_session_duration = 600   # 60 seconds is 10 minutes,
+                                auth_enabled = True)
+    ```
+
+=== "TOML configuration"
+
+    ```python linenums="1"
+    from taipy import Config
+
+    Config.load("config.toml")
+    ```
+
+    ```toml linenums="1" title="config.toml"
+
+    [TAIPY]
+
+    ldap_server="ldap://0.0.0.0"
+    ldap_base_dn="dc=example,dc=org"
+    secret_key = "my-ultra-secure-and-ultra-long-secret"
+    auth_session_duration = 600   # 60 seconds is 10 minutes,
+    auth_enabled = True
+    ```
+
+# Roles
+
+Taipy has four predefined user roles names that can be assigned to users. Each of these predefined roles provide a different set of capabilities and will be described in details below.
+
+!!! warning "Caution: Please note that Taipy does not provide functions to populate the predifined user roles to the user's LDAP server. The user is expected to populate the roles themselves."
```

Comment:
@FabienLelaquais I have added a small warning to let the user know about populating the roles to the LDAP and I would definitely like to hear your opinion on it :D

Reply:
That's neat.

The warning box however should be reformatted a little bit (typo fixed in 'predefined')
```
!!! warning
    Caution: Please note that Taipy does not provide functions to populate the predefined user roles to the user's
   LDAP server. The user is expected to populate the roles themselves."
```